app/Http/Controllers/MemberController.php

```diff
@@ -108,10 +117,19 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
+        $rules = [
```

Comment:
These are the same rules as above. To prevent duplication and a difference in validation for create and update, I think you should use a constant or class property for the validation rules.